docs/development/setting-up-environment.md

```diff
@@ -171,17 +222,18 @@ make livehtml
 
 You can visit the documentation server by opening a browser and visiting [http://127.0.0.1:8000](http://127.0.0.1:8000).
 
-* Alternately, you can open a static documentation server using the following command:
+* Alternately, you can open a static documentation server. Unlike the above, this will not automatically update any changes done after running this server. To see the changes done, you will need to manually stop and restart the server. To do this, use the following command:
+
 ```
 make htmlserve
 ```
 
 You can visit the documentation server by opening a browser and visiting [http://127.0.0.1:8080](http://127.0.0.1:8080).
 
-However, the above command will not automatically update any changes done after running this server. Any changes done will need the shutting down and restarting of the server.
-
 * To stop either server, press `ctrl+C` or `command+C` while the shell running the command is active.
 
+* Note: If the above make commands failed, you need to activate the documentation environment first before running any of the commands. Refer to [Activating the environment](#activating-documentation-environment) section above.
```

Comment:
I've been having trouble getting this anchor to work. I didn't want to use up too much time on this, so I uploaded as is.